src/TestImages.jl

```diff
@@ -1,10 +1,12 @@
 module TestImages
 using FileIO, AxisArrays
+if VERSION >= v"1.3"
+    using Pkg.Artifacts
+    const artifacts_toml = abspath(joinpath(@__DIR__, "..", "Artifacts.toml"))
+end
```

Comment:
This if statement can be deleted if we choose to drop `Julia v1.0` compatibility.